### PR TITLE
Add params unittest cases for cabundle

### PR DESCRIPTION
**Pull request by HumairAK**
Add unit tests for cabundle cases for params

### CODE REVIEW

File: controllers/testdata/tls/dummy-ca-bundle.crt [hunk 1]
```diff
@@ -0,0 +1 @@
+dummycontent
```
Comment (HumairAK) [inline controllers/testdata/tls/dummy-ca-bundle.crt:1]: ignore the no-newline here

**Comment by dsp-developers** [comment]
A new image has been built to help with testing out this PR: `quay.io/opendatahub/data-science-pipelines-operator:pr-616`
An OCP cluster where you are logged in as cluster admin is required.

To use this image run the following:

```bash
cd $(mktemp -d)
git clone git@github.com:opendatahub-io/data-science-pipelines-operator.git
cd data-science-pipelines-operator/
git fetch origin pull/616/head
git checkout -b pullrequest bbf9ea94175ef956cee433d374d87286cb0d29fd
oc new-project opendatahub
make deploy IMG="quay.io/opendatahub/data-science-pipelines-operator:pr-616"
```

More instructions [here](https://github.com/opendatahub-io/data-science-pipelines-operator#deploy-dsp-instance) on how to deploy and test a Data Science Pipelines Application.

**Comment by dsp-developers** [comment]
Change to PR detected. A new PR build was completed.
  A new image has been built to help with testing out this PR: `quay.io/opendatahub/data-science-pipelines-operator:pr-616`

**Review by rimolive — COMMENTED** [review]
/lgtm

**Pull request merged**